### PR TITLE
manifest,test: `inspect` should contain `OCI` annotations.

### PR DESCRIPTION
**Pull request by flouthoc**
Verifies https://github.com/containers/common/pull/1105 by adding a new test `inspect manifest and verify OCI annotation`
and old tests should not break.

#### Does this PR introduce a user-facing change?

```release-note
manifest: inspect now supports OCI annotations
```

### CODE REVIEW

**Comment by vrothberg** [comment]
> /hold this is just to verify [containers/common#1105](https://github.com/containers/common/pull/1105) so do not merge plz

Needs a test for exercising the new fields

**Comment by flouthoc** [comment]
> > /hold this is just to verify [containers/common#1105](https://github.com/containers/common/pull/1105) so do not merge plz
> 
> Needs a test for exercising the new fields

Added a test and https://github.com/containers/common/pull/1105 also contains a test. But this points to my fork so lets wait for `c/common` PR.

File: go.mod
Comment (TomSweeneyRedHat) [inline go.mod:15]: We should get a common release out and vendored before 8.7/9.1

**Review by vrothberg — COMMENTED** [review]
/lgtm
/hold

File: go.mod
Comment (vrothberg) [reply go.mod:15]: https://github.com/containers/common/pull/1107#pullrequestreview-1056811028

**Comment by vrothberg** [comment]
Tests seem to fail reproducibly.  I did not have a look at the failures though.

**Comment by flouthoc** [comment]
Bumped to `v0.49.1` instead of a commit in between lets see if CI is happy now or not.

**Comment by flouthoc** [comment]
I think this should be green now since `c/common` commit which i need is already here and PR https://github.com/containers/common/ also includes that commit which is already green: https://github.com/containers/podman/pull/15108 so if https://github.com/containers/podman/pull/15108 gets merged before, I'll drop vendor changes from here and will only add test.

File: go.mod
Comment (flouthoc) [reply go.mod]: bumped to `v0.49.1`

**Comment by flouthoc** [comment]
I think https://github.com/containers/podman/pull/15108 is needed before I can bump vendors here, @mtrmac Could you correct me if i am assuming this wrong.

**Comment by flouthoc** [comment]
Needed feature is already in here: https://github.com/containers/podman/pull/15108 so dropping vendor changes and just adding test.

**Review by vrothberg — COMMENTED** [review]
/lgtm
/hold

**Comment by flouthoc** [comment]
Is podman upstream broken ? Current PR does not adds anything other than a new test and some unrelated tests are failing.

All failing tests have common
```
Error: error storing signature for docker.io/library/alpine:latest, sha256:fa93b01658e3a5a1686dc3ae55f170d8de487006fb53a28efcd12ab0710a2e5f: Unusable secret key
```

**Comment by flouthoc** [comment]
Yes I see same failures in other PR's https://github.com/containers/podman/pull/15136

**Comment by vrothberg** [comment]
I don't really understand why.  Did the key expire? Cc: @Luap99

**Comment by vrothberg** [comment]
```
$ gpg --list-secret-keys            
[...] sec   rsa2048 2020-07-31 [SC] [expired: 2022-07-31] 
```

Yes, the key expired yesterday.

**Comment by vrothberg** [comment]
https://github.com/containers/podman/pull/15139 will fix it

**Comment by flouthoc** [comment]
> #15139 will fix it

@vrothberg thanks :) I'll rebase once #15139 gets merged.

**Comment by flouthoc** [comment]
This should become green now ( hopefully ).

**Comment by TomSweeneyRedHat** [comment]
LGTM
and happy greeen tests

**Review by vrothberg — APPROVED** [review]
/lgtm

**Pull request merged**
